WordPress/Classes/Utility/WebKitViewController.swift

```diff
@@ -359,9 +359,18 @@ class WebKitViewController: UIViewController, WebKitAuthenticatable {
 
     /// Sets the width of the web preview
     /// - Parameter width: The width value to set the webView to
-    func setWidth(_ width: CGFloat?) {
+    func setWidth(_ width: CGFloat?, viewSize: CGSize? = nil) {
         if let width = width {
-            widthConstraint?.constant = min(width, view.superview?.frame.width ?? width)
+            var horizontalViewBound: CGFloat
+            if let viewWidth = viewSize?.width {
+                horizontalViewBound = viewWidth
+            } else if let superViewWidth = view.superview?.frame.width {
```

Comment:
I realized that `view.superview?.frame.width` won't have the "going to" value from `viewWillTransition`, otherwise we could simplify things some. This makes sense. 👍

Reply:
I interpret your comment to mean no changes are needed here. Please correct me if I am mistaken, however.

Reply:
Yep! All good. 👍